asp_interactive_configuration/ooasp/tests/test_metro.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) 2022 Siemens AG Oesterreich
+# SPDX-License-Identifier: MIT
+
+from ooasp.interactive import InteractiveConfigurator
+from ooasp.kb import OOASPKnowledgeBase
+from ooasp import settings
+from ooasp.tests.utils import new_iconf
+
+import pytest
+from importlib import reload
+
```

Comment:
Several test_metro test cases are currently failing. The issue seems to be related to fclingo.

> FAILED ooasp/tests/test_metro.py::test_metro_next - TypeError: object of type 'int' has no len()
> FAILED ooasp/tests/test_metro.py::test_metro_options - TypeError: object of type 'int' has no len()
> FAILED ooasp/tests/test_metro.py::test_metro_cv_wagon_constraints - TypeError: object of type 'int' has no len()
> FAILED ooasp/tests/test_metro.py::test_metro_cv_wagon_wrong_nr_pass - TypeError: object of type 'int' has no len()
> FAILED ooasp/tests/test_metro.py::test_metro_cv_handrail_required - TypeError: object of type 'int' has no len()
> FAILED ooasp/tests/test_metro.py::test_metro_multi_wagon - TypeError: object of type 'int' has no len()
> FAILED ooasp/tests/test_metro.py::test_metro_cv_wagon - TypeError: object of type 'int' has no len()
> FAILED ooasp/tests/test_metro.py::test_metro_cv_seat_color - TypeError: object of type 'int' has no len()

Reply:
Yes, this is an issue with fclingo, since it is still on development, the new version broke things. I will fix the version and push this change.